doc/tutorial/conv_arithmetic.txt

```diff
@@ -0,0 +1,834 @@
+.. _conv_arithmetic:
+
+===============================
+Convolution arithmetic tutorial
+===============================
+
+.. note::
+
+    This tutorial is adapted from an existing `convolution arithmetic guide
+    <https://arxiv.org/abs/1603.07285>`_ [#]_, with an added emphasis on
+    Theano's interface.
+
+Quick reference
```

Comment:
I like the idea, but I think it would be better not to put it at the beginning of the tutorial, as users might get confused reading it. I would either put it at the end (it will be quickly accessible from the navigation bar) or in a separate page linked from this one, so that the experienced users can just have a reference to that page. Should I express a preference, I would vote for the former.